core/src/smartcontracts/isi/asset.rs

```diff
@@ -103,6 +103,45 @@ pub mod isi {
         }
     }
 
+    impl Execute for Transfer<Asset, Metadata, Account> {
+        #[metrics(+"transfer_store")]
+        fn execute(self, _authority: &AccountId, wsv: &mut WorldStateView) -> Result<(), Error> {
```

Comment:
Btw, maybe it can be implemented with combination of `Unregister` and `Register`?

Reply:
Check current implementation